src/test/java/io/jenkins/plugins/analysis/core/steps/IssuesRecorder_DescriptorTest.java

```diff
@@ -0,0 +1,96 @@
+package io.jenkins.plugins.analysis.core.steps;
+
+import org.junit.jupiter.api.Test;
+
+import io.jenkins.plugins.analysis.core.steps.IssuesRecorder.Descriptor;
+import static io.jenkins.plugins.analysis.core.testutil.FormValidationAssert.assertThat;
+
+import hudson.util.FormValidation;
+
+/**
```

Comment:
[Formatting Regeln ](https://github.com/uhafner/codingstyle)in der ganzen Klasse beachten.